app/components/primer/open_project/sub_header_element.ts

```diff
@@ -31,6 +53,37 @@ class SubHeaderElement extends HTMLElement {
 
     this.classList.remove('SubHeader--expandedSearch')
   }
+
+  #setupFilterInputClearButton() {
```

Comment:
Please do not use special characters in the name of a method.
```suggestion
  setupFilterInputClearButton() {
```

Reply:
This pattern was borrowed from the primer repository. Removed.